tools/mocha-runner/src/mocha-runner.ts

```diff
@@ -238,13 +228,7 @@ async function main() {
         'npx',
         [
           ...(useCoverage
-            ? [
-                'c8',
-                '--check-coverage',
-                '--lines',
-                String(suite.expectedLineCoverage),
-                'npx',
-              ]
+            ? ['c8', '--check-coverage', '--lines', '90', 'npx']
```

Comment:
How about we remove this all together, it's not a meaningful representation considering we have BiDi and CDP, implementation. We need to merge all the runs and the gather the coverage (Firefox vs  Chrome Launcher etc.).

Reply:
We can consider it later, for now, it is very similar behavior.